_ext/rdmkit_panel.py

```diff
@@ -0,0 +1,106 @@
+from os import path
+import yaml
+from yaml.loader import SafeLoader
+import json
+from docutils import nodes
+from docutils.parsers.rst import Directive
+from docutils.parsers.rst import directives
+import sphinx.errors
+import sphinx_panels.panels
+from sphinx.util import logging
+
+logger = logging.getLogger(__name__)
+logger.info('Hello, this is "rdmkit_panel" extension!')
+
+
+class RDMkitPanel(Directive):
+    has_content = False
+
+    def _create_content(self):
+        self.parse_yaml()
+        content = []
+        content.extend([
+            '<div class="card w-100 border-2 docutils">',
+                '<div class="card-header bg-primary pa_dark docutils">',
+                    '<a href="../../../_images/RDMkit_logo.svg" id="rdmkit-logo" class="reference internal image-reference" target="_blank">',
+                        '<img alt="../../../_images/RDMkit_logo.svg" height="40px" id="rdmkit-logo" src="../../../_images/RDMkit_logo.svg">',
+                    '</a>',
+                '</div>',
+                '<div class="card-body docutils">',
+                    self.get_rdmkit_html(),
+                '</div></div>'
+        ])
+        return content
+
+    def get_rdmkit_html(self):
+        """
+        Creates the rdmkit html snippets for each link
+        :return str: 
+        """
+        
+        for rdmkit_title, rdmkit_filename in self.fcb_rdmkit_links.items():
+            output = '<ul>'
+            output += f'<li><a href="https://rdmkit.elixir-europe.org/{rdmkit_filename}">General guidance about {rdmkit_title}</a></li>'
+            output += '</ul>'
+            return output
+
+
+    def parse_yaml(self):
+
+        here_path = path.dirname(path.abspath(__file__))
+        yaml_path = path.join(here_path, '..', '_static','faircookbook_rdmkit_mapping.yml')
+        json_path = path.join(here_path, '..', '_static', 'recipes.json')
+
+        with open(yaml_path, 'r') as f:
+            yaml_data = yaml.load(f, Loader=SafeLoader)
+        with open(json_path, 'r') as f:
+            json_data = json.load(f)
+
+        latest_recipe = list(json_data)[-1]
+        fcb_id = json_data[latest_recipe]['identifier']
+
+        self.fcb_rdmkit_links = {}
+
+        for rdmkit_page in yaml_data:
+            for fcb_link in rdmkit_page['links']:
+                if fcb_link['fcb_id'] == fcb_id:
+                    self.fcb_rdmkit_links[rdmkit_page['rdmkit_title']] = rdmkit_page['rdmkit_filename']
```

Comment:
Considering how the self.fcb_rdmkit_links array is being build, there could, in theory be more than one link.
In that case we want to change the get_rdmkit_html() methods so that the `<ul>` and return statement are outside the for loop at line 41.

Reply:
```suggestion
        output = '<ul>'
        for rdmkit_title, rdmkit_filename in self.fcb_rdmkit_links.items():
            output += f'<li><a href="https://rdmkit.elixir-europe.org/{rdmkit_filename}">General guidance about {rdmkit_title}</a></li>'
        output += '</ul>'
        return output


    def parse_yaml(self):

        here_path = path.dirname(path.abspath(__file__))
        yaml_path = path.join(here_path, '..', '_static','faircookbook_rdmkit_mapping.yml')
        json_path = path.join(here_path, '..', '_static', 'recipes.json')

        with open(yaml_path, 'r') as f:
            yaml_data = yaml.load(f, Loader=SafeLoader)
        with open(json_path, 'r') as f:
            json_data = json.load(f)

        latest_recipe = list(json_data)[-1]
        fcb_id = json_data[latest_recipe]['identifier']

        self.fcb_rdmkit_links = {}

        for rdmkit_page in yaml_data:
            for fcb_link in rdmkit_page['links']:
                if fcb_link['fcb_id'] == fcb_id:
                    self.fcb_rdmkit_links[rdmkit_page['rdmkit_title']] = rdmkit_page['rdmkit_filename']
```

Reply:
you are right! Thanks for spotting this!